rest_test.go

```diff
@@ -111,33 +88,25 @@ func TestRest(t *testing.T) {
 }
 
 func TestDefaultContentTypeWithBody(t *testing.T) {
-	host := "http://localhost"
-	method := Get
-	request := Request{
-		Method:  method,
-		BaseURL: host,
-		Body:    []byte("Hello World"),
-	}
-	response, _ := BuildRequestObject(request)
-	if response.Header.Get("Content-Type") != "application/json" {
-		t.Error("Content-Type not set to the correct default value when a body is set.")
+	fakeServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintln(w, "{\"message\": \"success\"}")
```

Comment:
The string in here should be:

```go
`{"message": "success"}`
```

(essentially, try to use a raw string any time your string has double quotes in it, to avoid all the quote-escaping)